.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cst"]
+	path = cst
+	url = git@github.com:balena-os/cst.git
```

Comment:
Can we please add the `branch` property here (master I assume)? It makes it easier to bump a submodule by hand, and allows Renovate to operate on non-master branches of submodules if we choose to do so in the future.